policy/Object.yaml

```diff
@@ -0,0 +1,381 @@
+funcs:
+    objectId: # the index of deepCoadd_obj IS the objectId
+        functor: Index
+    parentObjectId:
+        functor: Column
+        args: parent
+        dataset: ref
+    coord_ra:
+        # reference position for merged "ref" cat. Required by db. Not in DPDD
+        functor: RAColumn
+        dataset: ref
+    coord_dec:
+        # reference position for merged "ref" cat. Required by db. Not in DPDD
+        # coord_dec because "dec" is reserved in most SQL DBs
+        functor: DecColumn
+        dataset: ref
+    Ra:
+        functor: RAColumn
+        dataset: meas
+    Dec:
+        functor: DecColumn
+        dataset: meas
+    # raErr:  not available yet DM-15180
+    # decErr: not available yet DM-15180
+    refBand: # reference band is same for all measurements
+        functor: ReferenceBand
+        dataset: ref
+    # psRadecTai: No moving point source model yet.
+    # psRadec: No moving point source model yet.
+    # psPm: No moving point source model yet.
+    # psParallax: No moving point source model yet.
+    PsFlux:
+        functor: NanoJansky
+        filt: HSC-R
+        # Should we include forced_src too?
+        args: slot_PsfFlux_instFlux
+    PsFluxErr:
+        functor: NanoJanskyErr
+        dataset: meas
+        args:
+            - slot_PsfFlux_instFlux
+            - slot_PsfFlux_instFluxErr
+    # Assume the database will provide UDFs for Mags.
+    # psCov:
+    #     Flux should be somewhere in the name?
+    #     How best to store cov matrix?
+    # psLnL:  No moving point source model yet.
+    # psChi2:  No moving point source model yet.
+    refPsNdata:
+        # This does not make sense outside a moving point source model,
+        # because slot_PsfFlux_area is just the footprint on the coadd and depends on seeing
+        # At the moment it only makes sense to do this per band
+        # DPDD thinks this is an integer. It is not.
+        functor: Column
+        args: slot_PsfFlux_area
```

Comment:
What the DPDD wants isn't just the footprint area, because the pixels we use are defined by the size of the PSF model bounding box, not the footprint.  In any case, the larger point that it's not very meaningful when we're not doing a moving PS is certainly true.

However, `base_PsfFlux_area` isn't really related to any of these (and that's why it's not an integer); it's a measure of PSF width that has units of area rather than units of length (and sadly, "effective area" means something totally different from "effective radius").  That's a useful quantity, but not one in the DPDD at all.

Reply:
Is what the DPDD wants something that lets us go from Chi^2 to reduced Chi^2? I'm going to call `psNdata` "Not computed yet,"

Reply:
Yes, that's my understanding.